src/main/java/org/usfirst/frc4904/auton/Poses.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc4904.auton;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+
+class Poses {
```

Comment:
I'm not gonna bother with checking all of these constants right now, but just remember that - is right and clockwise, + is left and ccw.

Reply:
yeah most of these are probably wrong, especially the rotations